ui/utils/hooks/useNotification.js

```diff
@@ -61,39 +61,43 @@ export const useNotification = () => {
   }) => {
     timestamp = timestamp ?? moment.utc().valueOf();
     id = id || v4();
+    // Check if the notification center toggle class exists
+    const notificationCenterExists = document.querySelector(`.${NOTIFICATION_CENTER_TOGGLE_CLASS}`);
```

Comment:
the problem is not that events are getting duplicates , its only for specific events that are toasted manually from ui and also recieved from server .

Reply:
this logic will disable all ui toasts .

Reply:
Hello @aabidsofi19 Can you tell me what is the specific event